test/integration/GCHP/README.md

```diff
@@ -17,76 +17,76 @@ Integration tests are short GCHP simulations.  Executables are compiled with all
 ### Integration Test Scripts
 
 - `integrationTest.sh`
-  - Driver script to run GCHP integration tests
+  - Driver script to run GEOS-Chem Classic integration tests
 - `integrationTestCreate.sh`
-  - Script to create GCHP run directories for integration tests
+  - Script to create GEOS-Chem Classic run directories for integration tests
 - `integrationTestCompile.sh`
-  - Script to compile GCHP executables
+  - Script to compile GEOS-Chem Classic executables
 - `integrationTestExecute.sh`
-   - Script to run GCHP integration test simulations
+   - Script to run GEOS-Chem Classic integration test simulations
```

Comment:
These should be reverted to "GCHP" and not "GEOS-Chem Classic"

Reply:
@msulprizio: Now fixed in 2ea36ea8425237606cc69011473af7277d9412b9